### PR TITLE
feat(Components): Added basic Modal component

### LINKED ISSUE #48: Create dialog component
Skjermbilde 2020-09-20 kl  01 10 18

Maybe the Kit should have a dialog component, featuring the backdrop and very simple dialog implementation?

We need to do some research!

### PR DESCRIPTION
**Pull request by kvokov**
@cobraz I'm not sure about the exact styles, need  sources
image

closes #48

### CODE REVIEW

**Comment by cobraz** [comment]
We should add Netlify preview PRs, so that @AnneMatilde can review. I'll do that right away.

@AnneMatilde can you pitch in here? Maybe @kvokov can add a screenshot, since we don't have PR preview ATM.

**Comment by cobraz** [comment]
I've added PR preview. Since this PR was already open, I've built this branch manually.

Check the results here:
https://5f7f1571d5edf39e30683a5d--tabetalt-kit.netlify.app/

@AnneMatilde 👆

File: src/components/Modal/Modal.tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    backgroundColor: 'rgba(0, 0, 0, 0.1)',
+  },
+  content: {
+    position: 'relative',
+    transform: 'translateY(-50%)',
+    top: '50%',
+    width: '100%',
+    maxWidth: '70%',
+    maxHeight: 'auto',
+    minHeight: 'auto',
```
Comment (cobraz) [inline src/components/Modal/Modal.tsx:20]: Aren't these `auto` by default?

**Review by cobraz — APPROVED** [review]
LGTM 🎉

File: src/components/Modal/Modal.tsx (hunk 1 above)
Comment (kvokov) [reply src/components/Modal/Modal.tsx]: oh yeah
removed!

**Review by TanyaDolgopolova — APPROVED** [review]

**Comment by kvokov** [comment]
Updated PR with minor styles fix to match the design spec

**Comment by AnneMatilde** [comment]
This component looks good. Please change the font to "H2" and change the "close modal" button, like shown here:
Skjermbilde 2020-10-09 kl  16 45 17

The design here is also updated if you need to see more details. 
https://xd.adobe.com/view/87cd5089-53b2-43d4-7193-5a9849007b08-f1fc/
Password: TabetaltAdmin2020

**Comment by kvokov** [comment]
@AnneMatilde added an optional header and close button

**Pull request merged**